src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/Books';
+
+const reducer = combineReducers({
+  booksReducer,
+});
+
+const store = createStore(
+  reducer,
+  applyMiddleware(logger),
+);
+
+export default store;
```

Comment:
- [ ] Kindly Import the `categoriesRuducer` from `./categories/categories.js`

- [ ] `booksReducer` and `categoriesReducer` should be added to `combineReducers`.